internal/service/logbook.go

```diff
@@ -191,18 +244,20 @@ func (l *logbookService) GetLogbookEntries(userID uint, start, end time.Time) ([
 
 	flights, err := l.flightRepository.GetByUserIDAndDate(userID, start, end)
 	if err != nil {
-		return nil, err
+		return logbookResponses, err
 	}
 
-	for _, flight := range flights {
+	if len(flights) == 0 {
```

Comment:
Why? This line does nothing. At the bottom, you are actually already returning empty slice, as `for` does not do anything